docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -505,3 +505,56 @@ Solution: Write in the GROUP BY query SearchPhrase HAVING uniqUpTo(4)(UserID) >=
 ## sumMapFiltered(keys_to_keep)(keys, values) {#summapfilteredkeys-to-keepkeys-values}
 
 Same behavior as [sumMap](../../sql-reference/aggregate-functions/reference/summap.md#agg_functions-summap) except that an array of keys is passed as a parameter. This can be especially useful when working with a high cardinality of keys.
+
+## sequenceNextNode {#sequenceNextNode}
+
+Returns a value of next event that matched an event chain.
```

Comment:
Because events can have the same timestamp, it can't be strictly increasing, but monotonically increasing.

In this case, the ordered chain is:
```
('1970-01-01 09:00:01',1,'A');
('1970-01-01 09:00:01',1,'A');
('1970-01-01 09:00:02',...);
...omitted...
```
Because timestamp is the first ordering criteria.

So that it produces `'A'` is expected.

The omitted events can be clarified by applying your recommendation that `always choose smallest value comparing as strings.`
```
('1970-01-01 09:00:01',1,'A');
('1970-01-01 09:00:01',1,'A');
('1970-01-01 09:00:02',1,'B');
('1970-01-01 09:00:02',1,'C');
('1970-01-01 09:00:02',1,'X');
```

That's great. I will fix it in order to choose smallest value comparing as strings.